dnf5daemon-server/session.hpp

```diff
@@ -91,7 +91,7 @@ class Session {
     /// download packages for the current transaction
     void download_transaction_packages();
     /// prepare the current transaction to run during the next reboot
-    void store_transaction_offline();
+    void store_transaction_offline(bool downloadonly);
```

Comment:
Is a version bump needed if the `Session::store_transaction_offline` changed what arguments it takes? Just asking, because, I don't know what makes up the API for dnf5daemon.